scripts/bootstrap-ansible.sh

```diff
@@ -30,36 +30,40 @@ export ANSIBLE_ROLE_FETCH_MODE=${ANSIBLE_ROLE_FETCH_MODE:-git-clone}
 
 ## Main ----------------------------------------------------------------------
 
-# If the installation is an upgrade the $OS_DIR path will alredy exist from a
-#  previous submodule checkout. In the event that it does exist, this will move
-#  the directory to the proper location.
-if git config --file "${BASE_DIR}/.gitmodules" --name-only --get-regexp path | grep -q "openstack-ansible" && [[ -d "${OA_DIR}" ]]; then
-  mv "${OA_DIR}" /opt/openstack-ansible
-elif [[ ! -L "${BASE_DIR}/openstack-ansible" ]] && [[ -d "${BASE_DIR}/openstack-ansible/.git" ]]; then
-  mv "${OA_DIR}" /opt/openstack-ansible
-elif [[ ! -d "/opt/openstack-ansible" ]]; then
+# If /opt/openstack-ansible exists, delete it if it is not a git clone
+if [[ -d "/opt/openstack-ansible" ]] && [[ ! -d "/opt/openstack-ansible/.git" ]]; then
```

Comment:
for consistency, I'd replace all the /opt/openstack-ansible by
${OA_DIR}

because ${OA_DIR} is used later in the process.

Reply:
The value for ``OA_DIR`` is ``/opt/rpc-openstack/openstack-ansible``, not ``/opt/openstack-ansible``. Implementing that change over can be done by whoever merges the patch to switch all scripts from using the current value. For now, I'm simply re-implementing what was already there in terms of the use of this value.

Reply:
I am not sure to understand what will happen for L77/L81 then.

Reply:
L64 creates a symlink to ensure that all scripts worked as they did when the submodule was present.